k8s/trillian/manifest/logserver.yaml

```diff
@@ -0,0 +1,71 @@
+apiVersion: v1
+kind: List
+items:
+- apiVersion: v1
+  kind: Service
+  metadata:
+    name: $NAME-logserver-service
+    labels: &LogServerLabels
+      app.kubernetes.io/name: "$NAME"
+      app.kubernetes.io/component: logserver
+    annotations:
+        cloud.google.com/load-balancer-type: "Internal"
```

Comment:
This type of Service will not be portable across other K8s implementations than GKE. Are there any other requirements limiting the portability of the application to GKE?

Reply:
I don't believe so, no. I think this is still compatible with other K8s implementations, you just won't get this feature (which is a nice-to-have, rather than required to use Trillian).

Reply:
Yes, you are right - if this is not required for Trillian, we are fine with having this annotation here.